internal/future/statestore/redis.go

```diff
@@ -119,7 +120,72 @@ func (rb *redisBackend) DeindexTicket(context.Context, string, []string) error {
 	return status.Error(codes.Unimplemented, "not implemented")
 }
 
-// FilterTickets returns the Ticket ids for the Tickets meeting the specified filtering criteria.
-func (rb *redisBackend) FilterTickets(context.Context, []pb.Filter) ([]string, error) {
-	return nil, status.Error(codes.Unimplemented, "not implemented")
+// FilterTickets returns the Ticket ids and required attribute key-value pairs for the Tickets meeting the specified filtering criteria.
+// map[ticket.Id]map[attributeName][attributeValue]
```

Comment:
Can we also add a commented example? That would make this super clear - something like:
{
    "testplayer1":  {"ranking" : 56, "loyalty_level": 4},
    "testplayer2":  {"ranking" : 50, "loyalty_level": 3}
}